esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -845,6 +864,29 @@ void WsWuInfo::getHelpers(IEspECLWorkunit &info, unsigned long flags)
     }
 }
 
+#ifdef _CONTAINERIZED
+const char *WsWuInfo::getImportedComponentLog(StringBuffer &log)
+{
+    SCMStringBuffer s;
+    cw->getDebugValue("imported", s);
```

Comment:
minor: getDebugValueBool() would be more usual.

Reply:
The 'imported' is a comma separated string, not a Boolean.